gunicorn/workers/ggevent.py

```diff
@@ -87,6 +87,10 @@ def run(self):
             self.notify()
             gevent.sleep(1.0)
 
+        # Wait for pending keepalive connections to complete before stopping the acceptance of new requests
+        if self.cfg.keepalive:
```

Comment:
i disagree there, we shouldn't care about waiting for  keepalive connections when closing. CLose should be done as fast as possible.